pymilvus/orm/connections.py

```diff
@@ -254,6 +254,8 @@ def __get_full_address(
             return address, None
 
         if uri != "":
+            if isinstance(uri, str) and uri.startswith("unix:"):
+                return uri, None
```

Comment:
__get_full_address should return a str with format  `{host}:{port}`, uri startswiths "unix:" doesn't seems to be correct.

And why do you need a uri starts with "unix:" scheme? I don't know how is this possible for self-deployed Milvus. Could you please file an issue to describe in detail why is it needed?

Reply:
https://grpc.github.io/grpc/cpp/md_doc_naming.html
Using UDS (unix:path, unix://absolute_path – Unix domain sockets (Unix systems only)
) instead of ports resolves possible port conflicts